playground/mocks/config/responseConfig.js

```diff
@@ -115,6 +115,7 @@ const idx = {
   '/idp/idx/identify': [
     'authenticator-verification-email',
     // 'authenticator-enroll-select-authenticator',
+    // 'authenticator-verification-email-without-emailmagiclink',
```

Comment:
Just curious, what is this line commented out for?

Reply:
I added new json and added here if we want to test without emailmagiclink scenario so that everyone can see and use it rather than creating/searching for one.